flake.nix

```diff
@@ -150,7 +156,7 @@
     dream2nixFor = forAllSystems (system: pkgs:
       import ./src rec {
         externalDir = externalDirFor."${system}";
-        inherit externalPaths externalSources lib pkgs;
+        inherit all-cabal-json externalPaths externalSources lib pkgs;
```

Comment:
Maybe instead of directly adding this and any potential future inputs here, we could add it to `externalSources` instead? Or maybe just a new attribute called `sources`. Ideally we should not really add random inputs to any interface IMO since we want to port these to the module system, so we should either reuse `externalSources` or decide on another attribute. Besides I think it would make it less cluttered anyways.